scripts/ci/logcheck/check.sh

```diff
@@ -8,15 +8,20 @@
 DIR="$(cd "$(dirname "$0")" && pwd)"
 
 BLOCKLIST_FILE="${DIR}/blocklist-patterns"
-ALLOWLIST_FILE="${DIR}/allowlist-patterns"
+allow_file="${ALLOWLIST_FILE}"
+# Allow for tests to set this as an environment variable
+if [[ -z "${ALLOWLIST_FILE:-}" ]]; then
+    allow_file="${DIR}/allowlist-patterns"
+fi
```

Comment:
kind of a nit. (but if this script had `set -u` like most rox scripts the unbound variable access would cause a failure)
```suggestion
allow_file="${ALLOWLIST_FILE:-${DIR}/allowlist-patterns}"
```